cmode/cmode.go

```diff
@@ -54,7 +53,7 @@ func New(cmLogger CModeLogger, opts ...CModeOpt) CMode {
 	return cm
 }
 
-func Handler(cm CMode) http.Handler {
+func (cm CMode) Handler() http.Handler {
```

Comment:
```suggestion
func (cm *CMode) Handler() http.Handler {
```
Need to be consistent with receiver type (all methods by value or all methods by pointer, to avoid funny behavior with interfaces).